examples/draw-brush.c

```diff
@@ -8,10 +8,9 @@
 static HWND hwndMain = NULL;
 
 
-static const WD_COLOR drawColors[3] =
+static const WD_COLOR brushColors[3] =
         { WD_RGB(255,0,0), WD_RGB(0,255,0), WD_RGB(0,0,255) };
-static const WD_COLOR fillColors[3] =
-        { WD_ARGB(63,255,0,0), WD_ARGB(63,0,255,0), WD_ARGB(63,0,0,255) };
+static float stopOffsets[3] = { 0, 0.5f, 1.0f };
```

Comment:
Either `brushColors` & `brushOffsets`; or `stopColors` & `stopOffsets`, but not a mixture.

Reply:
stopColors would make more sense then IMO.